.github/linters/.protolintrc.yml

```diff
@@ -0,0 +1 @@
+../../.protolint.yaml
```

Comment:
I think this file is from a time before the linter presubmit, can probably just rename it and move it here

Reply:
done